### PR TITLE
Adding siuba

### PR DESCRIPTION
**Pull request by machow**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by machow** [comment]
Whew, I think we did it!

**Comment by machow** [comment]
Alright--I think it's ready real this time! Happy to make any changes--I think I saw some notes on pinging some groups of people when a PR is ready, so will look back to check!

File: recipes/siuba/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pandas >=0.24.0
+    - numpy >=1.12.0
+    - SQLALchemy >=1.1.18
```
Comment (marcelotrevisani) [inline recipes/siuba/meta.yaml]:
```suggestion
    - sqlalchemy >=1.1.18
```

File: recipes/siuba/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (marcelotrevisani) [inline recipes/siuba/meta.yaml]: Could you please remove these comments

File: recipes/siuba/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - siuba
+    - siuba.sql
```
Comment (marcelotrevisani) [inline recipes/siuba/meta.yaml]:
```suggestion
    - siuba.sql
  requires:
    - pip
  commands:
    - pip check
```

**Pull request closed without merging**